Radio.h

```diff
@@ -372,5 +377,6 @@ enum radioAtuState {
 - (void) cmdDeleteGlobalProfile:(NSString *)profile;                // Remove the global profile from the radio
 - (void) cmdSaveTxProfile:(NSString *)profile;                      // Save the current state as a transmit profile
 - (void) cmdDeleteTxProfile:(NSString *)profile;                    // Remove the transmit profile from the radio
+- (Cwx *) getCwx;                                                   // get a reference to the Cwx object
```

Comment:
Shouldn't cwx be a property of Radio just like the VitaManager?  Why a separate getter?